adal/src/telemetry/java/com/microsoft/aad/adal/EventStrings.java

```diff
@@ -186,6 +186,8 @@
 
     static final String ACQUIRE_TOKEN_WITH_REFRESH_TOKEN_2 = "5";
 
+    static final String ACQUIRE_TOKEN_WITH_SAML_ASSERTION = "20";    
```

Comment:
Does this match the API ID used by iOS? (Asking because I'm unsure)

Reply:
I checked the iOS corresponding [file](https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/blob/4dfd66fc0cf2b8aa8be3872d2ad36c5c90e1ee19/IdentityCore/src/telemetry/MSIDTelemetryEventStrings.m), they dont seem to be using API ID, also they dont have anything for saml assertion here. 

I'm not sure about this, what do you suggest we do?

Reply:
> I'm not sure about this, what do you suggest we do?

I looked it up, the API ID is 6. You can see it [here](https://github.com/AzureAD/azure-activedirectory-library-for-objc/blob/dev/ADAL/src/ADAuthenticationContext.m) on line 315.

Reply:
Brilliant; let's use the same value for Android and I can update the associated telemetry documentation on our end

Reply:
Doc updated; located [here ](https://microsoft.sharepoint.com/:w:/r/teams/aad/devex/_layouts/15/Doc.aspx?sourcedoc=%7BAAE38591-8A29-4A50-AE00-B9F66505B6D2%7D&file=Telemetry%20DDS.docx&action=default&mobileredirect=true)(corpnet access reqd)

Reply:
Have changed the API ID to 6, will be able to see the changes after the next commit. Will mark this conversation as resolved.